Towny/src/main/java/com/palmergames/bukkit/towny/confirmations/ConfirmationTransaction.java

```diff
@@ -17,30 +23,30 @@ public class ConfirmationTransaction {
 	 * A transaction which must succeed for a Confirmation to complete.
 	 *
 	 * @param costSupplier cost of the transaction. 
-	 * @param payee Account which will have to pay.
+	 * @param townyObject TownyObject which will have to pay.
 	 * @param loggedMessage The message logged in the money.csv file.
 	 * @param insufficientFundsMessage Transatable which will display the cannot pay message.
 	 */
-	public ConfirmationTransaction(Supplier<Double> costSupplier, Account payee, String loggedMessage, Translatable insufficientFundsMessage) {
+	public ConfirmationTransaction(Supplier<Double> costSupplier, TownyObject townyObject, String loggedMessage, Translatable insufficientFundsMessage) {
```

Comment:
This can just accept an EconomyHandler instance which resident/town/nation all implement